_data/simple-icons.json

```
@@ -3820,6 +3820,12 @@
             "hex": "ED2224",
             "source": "https://metal.equinix.com"
         },
+        {
+            "title": "Ericsson",
+            "hex": "0082f0",
```

```suggestion
            "hex": "0082F0",
```

Needs to be uppercase to keep the linter happy, apologies for the additional 0 in my original review...